venv/bin/activate

```diff
@@ -0,0 +1,76 @@
+# This file must be used with "source bin/activate" *from bash*
+# you cannot run it directly
+
+deactivate () {
+    # reset old environment variables
+    if [ -n "${_OLD_VIRTUAL_PATH:-}" ] ; then
+        PATH="${_OLD_VIRTUAL_PATH:-}"
+        export PATH
+        unset _OLD_VIRTUAL_PATH
+    fi
+    if [ -n "${_OLD_VIRTUAL_PYTHONHOME:-}" ] ; then
+        PYTHONHOME="${_OLD_VIRTUAL_PYTHONHOME:-}"
+        export PYTHONHOME
+        unset _OLD_VIRTUAL_PYTHONHOME
+    fi
+
+    # This should detect bash and zsh, which have a hash command that must
+    # be called to get it to forget past commands.  Without forgetting
+    # past commands the $PATH changes we made may not be respected
+    if [ -n "${BASH:-}" -o -n "${ZSH_VERSION:-}" ] ; then
+        hash -r
+    fi
+
+    if [ -n "${_OLD_VIRTUAL_PS1:-}" ] ; then
+        PS1="${_OLD_VIRTUAL_PS1:-}"
+        export PS1
+        unset _OLD_VIRTUAL_PS1
+    fi
+
+    unset VIRTUAL_ENV
+    if [ ! "${1:-}" = "nondestructive" ] ; then
+    # Self destruct!
+        unset -f deactivate
+    fi
+}
+
+# unset irrelevant variables
+deactivate nondestructive
+
+VIRTUAL_ENV="/home/arthur/VisualStudioProjects/Python/ranked-voting/venv"
```

Comment:
The `VIRTUAL_ENV` is hardcoded. It won't work in other paths like mine, I think.

Reply:
oh, ok it may not be the best or im not knowing how to set it up for everyone

Reply:
although, thanks.

Reply:
ill try to use the relative path, or ill delete it.

Reply:
i updated to relative path... still working

Reply:
I think if you create a `requirements.txt` and explain how to use a virtualenv with this file it would be better. :smile:

Reply:
ok, ill do it latter

Reply:
but is a common environment which allow everybody have the same packages, looks like a docker container, i putted the "how to use" in readme.md

Reply:
so, everything that i install in our "common python interpreter" will be available for everyone INSIDE the VENV, it means nobody ill have to config the setup, resulting in less missing time, helping people to fix broken installs

Reply:
try it on your machine, clone my fork read the readme, en try running it